app/index.html

```diff
@@ -325,6 +325,7 @@ <h4 class="modal-title" id="gcc-error-label" data-bind="html: errorWarningLabel"
         <!-- build:js scripts/vendor.js -->
         <!-- bower:js -->
         <script src="../bower_components/jquery/dist/jquery.js"></script>
+        <script src="../bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js"></script>
```

Comment:
That change (and the other jquery.notific8 one) keeps getting made automatically by something. This also happens to @jdtran23. I really don't know anything else about it, but I can remove the change if need be